kyo-zio/shared/src/main/scala/kyo/ZIOs.scala

```diff
@@ -46,5 +46,5 @@ object ZIOs:
         }
     end run
 
-    sealed private[kyo] trait GetZIO extends Effect[ZIO[Any, Nothing, *], Id]
+    sealed private[kyo] trait GetZIO extends ArrowEffect[ZIO[Any, Nothing, *], Id]
```

Comment:
Initially I was thinking this was a problem, but actually the context effects will be shifted into `Env` when we get line 20 working.